os/board/rtl8721csm/src/rtl8721csm_boot.c

```diff
@@ -214,6 +214,15 @@ void amebad_mount_partions(void)
 #endif /* CONFIG_FLASH_PARTITION */
 }
 
+#ifdef CONFIG_FTL_ENABLED
+extern const u8 ftl_phy_page_num;
+extern const u32 ftl_phy_page_start_addr;
+void app_ftl_init(void)
```

Comment:
If this function is used only in this file, please add static.

Reply:
I have added static.